templates/skaffold.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: skaffold/v2beta9
```

Comment:
Where is this `CustomResourceDefinition` created? Didn't see that in the infrastructure PR.

Reply:
Ah never mind, looks like this is not a k8s manifest, just a config file for skaffold.

Reply:
this is a skaffold configuraiton yaml, not k8s one.